### PR TITLE
CTV-2062: Add card config documentation for TruexAdRenderer Roku

### PR DESCRIPTION
**Pull request by episcopus**
This was a retrospective action item, adding a documentation page on the main docs repo for the TAR Roku choice card format.

This format stems from the legacy format used by Innovid and will later be migrated to a unified, cross-platform syntax, more or less [based on this](https://github.com/socialvibe/truex-ads-docs/blob/master/choice_card_config_params.md), as per what the HTML5 project is using.

I put in the documentation by doing a review of the latest TAR Roku codebase.

### CODE REVIEW

File: roku_tar_choice_card_config_params.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+| iy | n/a | y-coordinate of the interact button. |
+| iw | n/a | Width of the interact button. |
+| ih | n/a | Height of the interact button. |
+| ib | n/a | URL to sprite image for 'interactive ad' button. |
+| wx | n/a | x-coordinate of the watch button. |
+| wy | n/a | y-coordinate of the watch button. |
+| ww | n/a | Width of the watch button. |
+| wh | n/a | Height of the watch button. |
+| wb | n/a | URL to sprite image for 'watch video ads' button. |
+| bd | 3 | a value that specifies the number of seconds to delay when all buttons should be shown.  If not specified, there should be no delay. Only supported for video choice cards. |
```
Comment (kylelamx) [inline roku_tar_choice_card_config_params.md]: The [code](https://github.com/socialvibe/TruexAdRenderer-Roku/blob/develop/components/TruexChoiceCard.brs#L290) suggested that---
If not specified, the delay should take the default value (3 secs)

File: roku_tar_choice_card_config_params.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+| bg | n/a | URL for choice card background image. Used whem `bgv` is not present and turns on static choice card mode. |
+| bgv | n/a | URL for choice card background video. When present the choice card becomes a video choice card. |
+| vo | n/a | mp3 URL for choice card voice over audio file. Only supported for static choice cards. |
+| sk | n/a | URL of skip card image or mp4. Can be overridden at runtime by any branded cards sent down by the ad server. |
+| aa | 5 | Number of seconds for auto advance timer. Typically, 30 seconds. -1 for no auto advance. Skip cards need a positive value here. |
```
Comment (kylelamx) [inline roku_tar_choice_card_config_params.md]: The [code](https://github.com/socialvibe/TruexAdRenderer-Roku/blob/develop/components/TruexChoiceCard.brs#L456) suggested that the default value should be 30

**Review by kylelamx — COMMENTED** [review]
LGTM, but there are 2 issues with the default values

File: roku_tar_choice_card_config_params.md (hunk 1 above)
Comment (episcopus) [reply roku_tar_choice_card_config_params.md]: ok

File: roku_tar_choice_card_config_params.md (hunk 2 above)
Comment (episcopus) [reply roku_tar_choice_card_config_params.md]: ok - clarified that 5s is the default for skip cards, and 30s for choice cards

Comment (episcopus) [reply roku_tar_choice_card_config_params.md]: Also, it looks like we don't support -1 use case, so I'll remove this and create a JIRA.

Comment (kylelamx) [reply roku_tar_choice_card_config_params.md]: I think we do support no AA. but it is 0 instead of -1.

Comment (episcopus) [reply roku_tar_choice_card_config_params.md]: Well, I don't think on Roku it was supported - intentionally. Just looking at the source, there's no validation on the `aa` value except that it is not `invalid` and then the timer event just decrements this by 1, firing the opt out event if the resulting value is 0. So 0 might just result in a timer that's firing and continuously decrementing into the negative values, effectively achieving "disabled" timer. Although this is very much what I would call an unintentional behavior and I would probably prefer it to be undocumented until we really address this.

I'll create a 1.3 ticket for triage to consider.

**Comment by astein** [comment]
Looks okay to me.

**Review by kylelamx — APPROVED** [review]

**Pull request merged**